kafka_utils/util/zookeeper.py

```diff
@@ -234,9 +238,11 @@ def get_topics(
             partitions_data = {}
             for p_id, replicas in six.iteritems(topic_data['partitions']):
                 partitions_data[p_id] = {}
+                # Fetch partition-info from zookeeper
+                partition_info = self._fetch_partition_info(topic_id, p_id)
```

Comment:
Looks like we will be fetching partition state on every call of get_topics which can slow-down the call. We should only fetch-partition-state when required.